src/appdev/ApplicationAPI.js

```diff
@@ -49,6 +50,14 @@ class ApplicationAPI {
         this.express.use(this.getPath() + version, router);
       });
     });
+
+    // // Setup Firebase Admin
+    console.log('here');
+    console.log(process.env.FCM_AUTH_KEY_PATH);
+    admin.initializeApp({
```

Comment:
i think we should move the admin.initializeApp into the server.js file so that it gets initialized right when we create the app itself